backend/plugins/tourism_api/src/modules/pms/db/models/Branch.ts

```
@@ -0,0 +1,81 @@
+    }
+
+    public static async get(query: any) {
+      const branch = await models.PmsBranch.findOne(query).lean();
+
+      if (!branch) {
+        throw new Error('Bms Branch not found');
+      }
+      return branch;
+    }
+
+    public static generateToken(length: number = 32) {
```

ESLint

[error] Type number trivially inferred from a number literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)